acceptance/analyzer_test.go

```diff
@@ -114,11 +114,29 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 	})
 
 	when("group path is provided", func() {
+		var copyDir, containerName, cacheVolume string
+
+		it.Before(func() {
+			containerName = "test-container-" + h.RandString(10)
+			var err error
+			copyDir, err = ioutil.TempDir("", "test-docker-copy-")
+			h.AssertNil(t, err)
+		})
```

Comment:
IIRC we are adding this setup because if `h.Run()` fails, the test could exit before we've had a chance to run cleanup.

Reply:
You're right!
I wrote it as part of my last commit message.